packages/app/src/pages/endpoints/[endpointId]/_/content/types/table/index.tsx

```diff
@@ -174,3 +205,143 @@ const Operations: React.FC<OperationsProps> = ({
     </>
   );
 };
+
+const RowObject: React.FC<
+  BaseProps & { rowObject: Data; columns: TableColumn[]; close: () => void }
+> = ({ on, rowObject, columns, close }) => {
+  return (
+    <div className="flex flex-col h-full w-full">
+      <div className="px-10 pt-10">
+        <div className="text-right">
```

Comment:
Can you pass text-right to Button's className props?
It's not nice to have unnecessary divs.

Reply:
I will  pass text-right to Button's className props.